testing/builtin_hash_fct_test.py

```diff
@@ -19,7 +19,7 @@ def test_it(self):
         self.assertEqual(res[1][1], 9281407066453831864)
 
         res = mldb.query("SELECT hash({a: 12, b: 'coco'})")
-        self.assertEqual(res[1][1], 12351211796938911870)
+        self.assertEqual(res[1][1], 17779398697240081926)
```

Comment:
No.  Previously there was a bug where timestamps were taken into account in the hash.  The timestamp included was wrong due to a different bug which was fixed in this branch.  Once fixed, the hash changed; I modified the code such that it shouldn't ever change any more.

If we do have some code that is deployed that relies on hashes, we will need to plan how to recover from that.